ogc/src/main/scala/geotrellis/server/ogc/style/InterpolatedColorMap.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.server.ogc.style
+
+import geotrellis.raster.render._
+import geotrellis.raster.{RGBA => _, _}
+
+import scala.math._
+import java.util.Arrays.binarySearch
+
+
+case class InterpolatedColorMap(
```

Comment:
Should this be a part of GeoTrellis?

Reply:
I think that's a great idea but one which should be done as part of a larger push to handle common rendering strategies; do you have some thoughts about how best to begin working these changes in?

Reply:
@moradology at least to create an issue probably to think about it :+1: and you're probably the person who knows better how to describe it / how to bring it into GT. But it in general looked for me like a functionality that can be required in other projects as well (like RF).